recipes/stackvana-meas_algorithms/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "stackvana-meas_algorithms" %}
+# LSST DM versions are prefixed with letters
+#
+#  - a weekly build is 'w_2018_50'
+#  - a major release is 'v18_1'
+#
+# In order to play nice with conda, we take the following conventions
+#
+#  - for a weekly build 'w_2018_50', the conda version is '0.2018.50'
+#  - for a major version 'v18_1', the conda version is '18.1.0'
+#
+{% set version = "0.2023.49" %}
+{% set patch_version = version.split(".")[-1] if version.split(".")[-1].__len__() == 2 else "0" + version.split(".")[-1] %}
+{% set dm_tag = "w_" + version.split(".")[1] + "_" + patch_version if version.startswith("0.") else "v" + version.replace(".", "_") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://eups.lsst.codes/stack/src/tags/{{ dm_tag }}.list
+  sha256: 5f2ff69dfcd65134835f296bd227d3dc3ef8a80c8e6d3aa10d2f45b486cf944f
+
+build:
+  skip: true  # [win or py != 311]
+  number: 0
+  merge_build_host: true
+  script:
+    - stackvana-build meas_algorithms
+
+requirements:
+  host:
+    - stackvana-core =={{ version }}
+    - stackvana-afw =={{ version }}
+    - python
+  run:
+    - python
+    - stackvana-core =={{ version }}
+    - stackvana-afw =={{ version }}
+
+test:
+  commands:
+    - setup meas_algorithms
+    - python -c "import lsst.afw"
+
+about:
+  home: https://github.com/beckermr/stackvana-core
```

Comment:
It's hard for me to review this recipe when this "home" page is not a public repository, and the source download is just a plain text list of packages?

Reply:
The source is public. The list of packages is a set of eups requirements that is fed to EUPS and built.

Reply:
The versions are defined by this list which is why that list is the source. This list gets updated which is when you know a new version exists. This triggers the bot.